docker-compose-cluster.yml

```diff
@@ -8,18 +8,18 @@ services:
     container_name: 'ziggurat_rabbitmq'
 
   zookeeper:
-    image: zookeeper:3.4.9
+    image: confluentinc/cp-zookeeper:5.5.0
     hostname: zookeeper
     ports:
       - "2181:2181"
     environment:
-      ZOO_MY_ID: 1
-      ZOO_PORT: 2181
-      ZOO_SERVERS: server.1=zookeeper:2888:3888
-      ZOO_TICK_TIME: 2000
+      ZOOKEEPER_CLIENT_PORT: 2181
+      KAFKA_OPTS: "-Djava.security.auth.login.config=/etc/kafka/zookeeper_server_jaas.conf
+                -Dzookeeper.authProvider.1=org.apache.zookeeper.server.auth.SASLAuthenticationProvider
+                -Dzookeeper.allowSaslFailedClients=true
+                -Dzookeeper.requireClientAuthScheme=sasl"
     volumes:
-      - /tmp/ziggurat_kafka_cluster_data/zookeeper/data:/data
```

Comment:
why have we removed these. ?

Reply:
seemed redundant to mount this volume, will add back